app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -45,9 +47,4 @@ def identifier
   def selected_school_urn
     auth_hash.dig('extra', 'raw_info', 'organisation', 'urn')
   end
-
-  def log_debug_information
-    anonymised_email = identifier.gsub(/(.)./, '\1*')
```

Comment:
Anonymising this email wouldn't have been useful in the cases we've had so far. We need to find the exact email address of a person looking to sign in to confirm they are the ones requesting support, favouring the DSI ID will keep it completely anonymous in our logs and the full ID value can only be used with access to DSI support console.